### PR TITLE
Change where python3 is sourced from

### PR DESCRIPTION
**Pull request by egede**

### CODE REVIEW

**Comment by egede** [comment]
OK, this change seems to work. It changes the LHCb testing to use the same python3 installation as is used for the actual LHCb installation in /cvmfs. The reason that the testing stopped working was that the LHCb login made changes to the virtualenv.

File: ganga/GangaLHCb/test/testrunner [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 source /cvmfs/lhcb.cern.ch/lib/LbEnv && source LbLogin.sh
 export GANGA_CONFIG_PATH=GangaLHCb/LHCb.ini
-virtualenv -p python3 venv
-rm -f venv/bin/python
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/cvmfs/sft.cern.ch/lcg/releases/LCG_92python3/Python/3.6.3/x86_64-slc6-gcc62-opt/lib
```
Comment (mesmith75) [inline ganga/GangaLHCb/test/testrunner]: Might be worth using a more recent release and with centos7

**Review by mesmith75 — CHANGES_REQUESTED** [review]

**Comment by mesmith75** [comment]
The remaining issue is that the environment guessing doesn't work with python3. `os.environ` now returns an `os._environ` class object rather than just a dict so trying to `eval` the string into a dict doesn't work.

**Comment by egede** [comment]
If you can fix it, please go ahead.

**Review by mesmith75 — APPROVED** [review]

**Comment by mesmith75** [comment]
@egede I think that is ok

**Comment by egede** [comment]
Thanks for fixing this.

**Pull request merged**